src/meson.build

```diff
@@ -162,6 +162,8 @@ modflow_sources = files(
     'Model' / 'GroundWaterFlow' / 'gwf-uzf.f90',
     'Model' / 'GroundWaterFlow' / 'gwf-vsc.f90',
     'Model' / 'GroundWaterFlow' / 'gwf-wel.f90',
+    'Model' / 'GroundWaterFlow' / 'submodules' / 'sm-gwf-sfr-steady.f90',
```

Comment:
If these are already in a folder named submodules, do they need an "sm" prefix in the file name?  Maybe we can drop the "sm"?